client/blocks/reader-recommended-sites/style.scss

```diff
@@ -0,0 +1,158 @@
+// Custom breakpoints needed to match Related Posts
+
+$reader-related-card-v2-breakpoint-medium: "( min-width: 661px ) and ( max-width: 730px )";
+$reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
+
+.design.main {
+	padding: 0; // ### Temporary
```

Comment:
Added it because Devdocs has that weird extra padding, but it's been removed in: https://github.com/Automattic/wp-calypso/pull/13472/commits/8922b8257e13693b996004f04805c2569baad0cb